packages/core/src/utilities/VoxelManager.ts

```diff
@@ -129,7 +129,7 @@ export default class VoxelManager<T> {
    */
   public setAtIndex = (index, v) => {
     const changed = this._set(index, v);
-    if (changed) {
+    if (changed !== false) {
```

Comment:
The response is optional for the set method, as some set methods have no way to determine changed state - for those ones, assume it is changed always.